virtool/setup/handlers.py

```diff
@@ -214,4 +210,11 @@
 
     req.app["events"]["restart"].set()
 
-    return web.Response(status=302, headers={"Location": "/"})
+    template = Template(filename=os.path.join(sys.path[0], "templates", "setup_restart.html"))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mako templates allow HTML/JS rendering by default and are inherently open to XSS attacks. Ensure variables in all templates are properly sanitized via the 'n', 'h' or 'x' flags (depending on context). For example, to HTML escape the variable 'data' do ${ data |h }.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4918216)